### DISCUSSION TITLE
How to emit a event in moleculer?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [moleculerjs/moleculer#1189] [Q&A] [answered]
How to emit a event in moleculer? 

When I read this [document](https://moleculer.services/zh/docs/0.14/context.html):

> When you call an action or emit an event, the broker creates a Context instance that contains all request information and passes it to the action/event handler as a single argument.

I know `call an action` but in a micro service how to `emit an event`? what's the difference between `call an action`?

**Comment by icebob** [comment, marked as answer]
Emitting events: https://moleculer.services/docs/0.14/events.html
